#! /bin/sh /usr/share/dpatch/dpatch-run
## 0007_manpages_section.dpatch by  <fabbe@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Put man pages in correct section. 

@DPATCH@
diff -urNad trunk~/man/sasl.3 trunk/man/sasl.3
--- trunk~/man/sasl.3	2006-05-29 22:52:46.000000000 +0300
+++ trunk/man/sasl.3	2006-07-31 17:59:29.000000000 +0300
@@ -38,7 +38,7 @@
 .\" AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING
 .\" OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 .\" 
-.TH SASL "10 July 2001" SASL "SASL man pages"
+.TH SASL 3 "10 July 2001" SASL "SASL man pages"
 .SH NAME
 SASL \- SASL authentication library
 .SH DESCRIPTION
diff -urNad trunk~/man/sasl_authorize_t.3 trunk/man/sasl_authorize_t.3
--- trunk~/man/sasl_authorize_t.3	2006-05-29 22:52:46.000000000 +0300
+++ trunk/man/sasl_authorize_t.3	2006-07-31 17:59:29.000000000 +0300
@@ -38,7 +38,7 @@
 .\" AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING
 .\" OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 .\" 
-.TH sasl_authorize_t "10 July 2001" SASL "SASL man pages"
+.TH sasl_authorize_t 3 "10 July 2001" SASL "SASL man pages"
 .SH NAME
 sasl_authorize_t \- The SASL authorization callback
 
diff -urNad trunk~/man/sasl_auxprop.3 trunk/man/sasl_auxprop.3
--- trunk~/man/sasl_auxprop.3	2006-05-29 22:52:46.000000000 +0300
+++ trunk/man/sasl_auxprop.3	2006-07-31 17:59:29.000000000 +0300
@@ -38,7 +38,7 @@
 .\" AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING
 .\" OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 .\" 
-.TH sasl_auxprop "10 July 2001" SASL "SASL man pages"
+.TH sasl_auxprop 3 "10 July 2001" SASL "SASL man pages"
 .SH NAME
 sasl_auxprop \- How to work with SASL auxiliary properties
 
diff -urNad trunk~/man/sasl_auxprop_getctx.3 trunk/man/sasl_auxprop_getctx.3
--- trunk~/man/sasl_auxprop_getctx.3	2006-05-29 22:52:46.000000000 +0300
+++ trunk/man/sasl_auxprop_getctx.3	2006-07-31 17:59:29.000000000 +0300
@@ -38,7 +38,7 @@
 .\" AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING
 .\" OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 .\" 
-.TH sasl_auxprop_getctx "10 July 2001" SASL "SASL man pages"
+.TH sasl_auxprop_getctx 3 "10 July 2001" SASL "SASL man pages"
 .SH NAME
 sasl_auxprop_getctx \- Acquire an auxiliary property context
 
diff -urNad trunk~/man/sasl_auxprop_request.3 trunk/man/sasl_auxprop_request.3
--- trunk~/man/sasl_auxprop_request.3	2006-05-29 22:52:46.000000000 +0300
+++ trunk/man/sasl_auxprop_request.3	2006-07-31 17:59:29.000000000 +0300
@@ -38,7 +38,7 @@
 .\" AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING
 .\" OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 .\" 
-.TH sasl_auxprop_request "10 July 2001" SASL "SASL man pages"
+.TH sasl_auxprop_request 3 "10 July 2001" SASL "SASL man pages"
 .SH NAME
 sasl_auxprop_request \- Request Auxiliary Properties from SASL
 
diff -urNad trunk~/man/sasl_callbacks.3 trunk/man/sasl_callbacks.3
--- trunk~/man/sasl_callbacks.3	2006-05-29 22:52:46.000000000 +0300
+++ trunk/man/sasl_callbacks.3	2006-07-31 18:05:39.000000000 +0300
@@ -38,7 +38,7 @@
 .\" AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING
 .\" OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 .\" 
-.TH sasl_callbacks "12 February 2006" SASL "SASL man pages"
+.TH sasl_callbacks 3 "12 February 2006" SASL "SASL man pages"
 .SH NAME
 sasl_callbacks \- How to work with SASL callbacks
 
diff -urNad trunk~/man/sasl_canon_user_t.3 trunk/man/sasl_canon_user_t.3
--- trunk~/man/sasl_canon_user_t.3	2006-05-29 22:52:46.000000000 +0300
+++ trunk/man/sasl_canon_user_t.3	2006-07-31 17:59:29.000000000 +0300
@@ -38,7 +38,7 @@
 .\" AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING
 .\" OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 .\" 
-.TH sasl_server_userdb_checkpass_t "10 July 2001" SASL "SASL man pages"
+.TH sasl_server_userdb_checkpass_t 3 "10 July 2001" SASL "SASL man pages"
 .SH NAME
 sasl_server_userdb_checkpass_t \- Plaintext Password Verification Callback
 
diff -urNad trunk~/man/sasl_chalprompt_t.3 trunk/man/sasl_chalprompt_t.3
--- trunk~/man/sasl_chalprompt_t.3	2006-05-29 22:52:46.000000000 +0300
+++ trunk/man/sasl_chalprompt_t.3	2006-07-31 17:59:29.000000000 +0300
@@ -38,7 +38,7 @@
 .\" AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING
 .\" OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 .\" 
-.TH sasl_chalprompt_t "10 July 2001" SASL "SASL man pages"
+.TH sasl_chalprompt_t 3 "10 July 2001" SASL "SASL man pages"
 .SH NAME
 sasl_chalprompt_t \- Realm Acquisition Callback
 
diff -urNad trunk~/man/sasl_checkapop.3 trunk/man/sasl_checkapop.3
--- trunk~/man/sasl_checkapop.3	2006-05-29 22:52:46.000000000 +0300
+++ trunk/man/sasl_checkapop.3	2006-07-31 17:59:29.000000000 +0300
@@ -38,7 +38,7 @@
 .\" AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING
 .\" OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 .\" 
-.TH sasl_checkapop "29 June 2001" SASL "SASL man pages"
+.TH sasl_checkapop 3 "29 June 2001" SASL "SASL man pages"
 .SH NAME
 sasl_checkapop \- Check an APOP challenge/response
 .SH SYNOPSIS
diff -urNad trunk~/man/sasl_checkpass.3 trunk/man/sasl_checkpass.3
--- trunk~/man/sasl_checkpass.3	2006-05-29 22:52:46.000000000 +0300
+++ trunk/man/sasl_checkpass.3	2006-07-31 17:59:29.000000000 +0300
@@ -38,7 +38,7 @@
 .\" AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING
 .\" OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 .\" 
-.TH sasl_checkpass "10 July 2001" SASL "SASL man pages"
+.TH sasl_checkpass 3 "10 July 2001" SASL "SASL man pages"
 .SH NAME
 sasl_checkpass \- Check a plaintext password
 .SH SYNOPSIS
diff -urNad trunk~/man/sasl_client_init.3 trunk/man/sasl_client_init.3
--- trunk~/man/sasl_client_init.3	2006-05-29 22:52:46.000000000 +0300
+++ trunk/man/sasl_client_init.3	2006-07-31 17:59:29.000000000 +0300
@@ -38,7 +38,7 @@
 .\" AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING
 .\" OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 .\" 
-.TH sasl_client_init "21 June 2001" SASL "SASL man pages"
+.TH sasl_client_init 3 "21 June 2001" SASL "SASL man pages"
 .SH NAME
 sasl_client_init \- SASL client authentication initialization
 
diff -urNad trunk~/man/sasl_client_new.3 trunk/man/sasl_client_new.3
--- trunk~/man/sasl_client_new.3	2006-05-29 22:52:46.000000000 +0300
+++ trunk/man/sasl_client_new.3	2006-07-31 17:59:29.000000000 +0300
@@ -38,7 +38,7 @@
 .\" AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING
 .\" OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 .\" 
-.TH sasl_client_new "21 June 2001" SASL "SASL man pages"
+.TH sasl_client_new 3 "21 June 2001" SASL "SASL man pages"
 .SH NAME
 sasl_client_new \- Create a new client authentication object
 
diff -urNad trunk~/man/sasl_client_start.3 trunk/man/sasl_client_start.3
--- trunk~/man/sasl_client_start.3	2006-05-29 22:52:46.000000000 +0300
+++ trunk/man/sasl_client_start.3	2006-07-31 17:59:29.000000000 +0300
@@ -38,7 +38,7 @@
 .\" AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING
 .\" OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 .\" 
-.TH sasl_client_start "10 July 2001" SASL "SASL man pages"
+.TH sasl_client_start 3 "10 July 2001" SASL "SASL man pages"
 .SH NAME
 sasl_client_start \- Begin an authentication negotiation
 .SH SYNOPSIS
diff -urNad trunk~/man/sasl_client_step.3 trunk/man/sasl_client_step.3
--- trunk~/man/sasl_client_step.3	2006-05-29 22:52:46.000000000 +0300
+++ trunk/man/sasl_client_step.3	2006-07-31 17:59:29.000000000 +0300
@@ -38,7 +38,7 @@
 .\" AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING
 .\" OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 .\" 
-.TH sasl_client_step "10 July 2001" SASL "SASL man pages"
+.TH sasl_client_step 3 "10 July 2001" SASL "SASL man pages"
 .SH NAME
 sasl_client_step \- Perform a step in the authentication negotiation
 
diff -urNad trunk~/man/sasl_decode.3 trunk/man/sasl_decode.3
--- trunk~/man/sasl_decode.3	2006-05-29 22:52:46.000000000 +0300
+++ trunk/man/sasl_decode.3	2006-07-31 17:59:29.000000000 +0300
@@ -38,7 +38,7 @@
 .\" AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING
 .\" OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 .\" 
-.TH sasl_decode "10 July 2001" SASL "SASL man pages"
+.TH sasl_decode 3 "10 July 2001" SASL "SASL man pages"
 .SH NAME
 sasl_decode \- Decode data received
 
diff -urNad trunk~/man/sasl_dispose.3 trunk/man/sasl_dispose.3
--- trunk~/man/sasl_dispose.3	2006-05-29 22:52:46.000000000 +0300
+++ trunk/man/sasl_dispose.3	2006-07-31 17:59:29.000000000 +0300
@@ -38,7 +38,7 @@
 .\" AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING
 .\" OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 .\" 
-.TH sasl_dispose "10 July 2001" SASL "SASL man pages"
+.TH sasl_dispose 3 "10 July 2001" SASL "SASL man pages"
 .SH NAME
 sasl_dispose \- Dispose of a SASL connection object
 
diff -urNad trunk~/man/sasl_done.3 trunk/man/sasl_done.3
--- trunk~/man/sasl_done.3	2006-05-29 22:52:46.000000000 +0300
+++ trunk/man/sasl_done.3	2006-07-31 17:59:29.000000000 +0300
@@ -38,7 +38,7 @@
 .\" AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING
 .\" OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 .\" 
-.TH sasl_done "10 July 2001" SASL "SASL man pages"
+.TH sasl_done 3 "10 July 2001" SASL "SASL man pages"
 .SH NAME
 sasl_done \- Dispose of a SASL connection object
 
diff -urNad trunk~/man/sasl_encode.3 trunk/man/sasl_encode.3
--- trunk~/man/sasl_encode.3	2006-05-29 22:52:46.000000000 +0300
+++ trunk/man/sasl_encode.3	2006-07-31 17:59:29.000000000 +0300
@@ -38,7 +38,7 @@
 .\" AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING
 .\" OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 .\" 
-.TH sasl_encode "10 July 2001" SASL "SASL man pages"
+.TH sasl_encode 3 "10 July 2001" SASL "SASL man pages"
 .SH NAME
 sasl_encode \- Encode data for transport to authenticated host
 
diff -urNad trunk~/man/sasl_encodev.3 trunk/man/sasl_encodev.3
--- trunk~/man/sasl_encodev.3	2006-05-29 22:52:46.000000000 +0300
+++ trunk/man/sasl_encodev.3	2006-07-31 17:59:29.000000000 +0300
@@ -38,7 +38,7 @@
 .\" AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING
 .\" OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 .\" 
-.TH sasl_encode "10 July 2001" SASL "SASL man pages"
+.TH sasl_encode 3 "10 July 2001" SASL "SASL man pages"
 .SH NAME
 sasl_encode \- Encode data for transport to authenticated host
 
diff -urNad trunk~/man/sasl_errdetail.3 trunk/man/sasl_errdetail.3
--- trunk~/man/sasl_errdetail.3	2006-05-29 22:52:46.000000000 +0300
+++ trunk/man/sasl_errdetail.3	2006-07-31 17:59:29.000000000 +0300
@@ -38,7 +38,7 @@
 .\" AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING
 .\" OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 .\" 
-.TH sasl_errdetail "10 July 2001" SASL "SASL man pages"
+.TH sasl_errdetail 3 "10 July 2001" SASL "SASL man pages"
 .SH NAME
 sasl_errdetail \- Retrieve detailed information about an error
 
diff -urNad trunk~/man/sasl_errors.3 trunk/man/sasl_errors.3
--- trunk~/man/sasl_errors.3	2006-05-29 22:52:46.000000000 +0300
+++ trunk/man/sasl_errors.3	2006-07-31 17:59:29.000000000 +0300
@@ -38,7 +38,7 @@
 .\" AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING
 .\" OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 .\" 
-.TH sasl_errors "10 July 2001" SASL "SASL man pages"
+.TH sasl_errors 3 "10 July 2001" SASL "SASL man pages"
 .SH NAME
 sasl_errors \- SASL error codes
 
diff -urNad trunk~/man/sasl_errstring.3 trunk/man/sasl_errstring.3
--- trunk~/man/sasl_errstring.3	2006-05-29 22:52:46.000000000 +0300
+++ trunk/man/sasl_errstring.3	2006-07-31 17:59:29.000000000 +0300
@@ -38,7 +38,7 @@
 .\" AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING
 .\" OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 .\" 
-.TH sasl_errstring "10 July 2001" SASL "SASL man pages"
+.TH sasl_errstring 3 "10 July 2001" SASL "SASL man pages"
 .SH NAME
 sasl_errstring \- Translate a SASL return code to a human-readable form
 
diff -urNad trunk~/man/sasl_getconfpath_t.3 trunk/man/sasl_getconfpath_t.3
--- trunk~/man/sasl_getconfpath_t.3	2006-05-29 22:52:46.000000000 +0300
+++ trunk/man/sasl_getconfpath_t.3	2006-07-31 17:59:29.000000000 +0300
@@ -38,7 +38,7 @@
 .\" AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING
 .\" OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 .\" 
-.TH sasl_getconfpath_t "12 February 2006" SASL "SASL man pages"
+.TH sasl_getconfpath_t 3 "12 February 2006" SASL "SASL man pages"
 .SH NAME
 sasl_getconfpath_t \- The SASL callback to indicate location of the config files
 
diff -urNad trunk~/man/sasl_getopt_t.3 trunk/man/sasl_getopt_t.3
--- trunk~/man/sasl_getopt_t.3	2006-05-29 22:52:46.000000000 +0300
+++ trunk/man/sasl_getopt_t.3	2006-07-31 17:59:29.000000000 +0300
@@ -38,7 +38,7 @@
 .\" AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING
 .\" OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 .\" 
-.TH sasl_getopt_t "10 July 2001" SASL "SASL man pages"
+.TH sasl_getopt_t 3 "10 July 2001" SASL "SASL man pages"
 .SH NAME
 sasl_getopt_t \- The SASL get option callback
 
diff -urNad trunk~/man/sasl_getpath_t.3 trunk/man/sasl_getpath_t.3
--- trunk~/man/sasl_getpath_t.3	2006-05-29 22:52:46.000000000 +0300
+++ trunk/man/sasl_getpath_t.3	2006-07-31 17:59:29.000000000 +0300
@@ -38,7 +38,7 @@
 .\" AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING
 .\" OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 .\" 
-.TH sasl_getpath_t "10 July 2001" SASL "SASL man pages"
+.TH sasl_getpath_t 3 "10 July 2001" SASL "SASL man pages"
 .SH NAME
 sasl_getpath_t \- The SASL callback to indicate location of the mechanism drivers
 
diff -urNad trunk~/man/sasl_getprop.3 trunk/man/sasl_getprop.3
--- trunk~/man/sasl_getprop.3	2006-05-29 22:52:46.000000000 +0300
+++ trunk/man/sasl_getprop.3	2006-07-31 17:59:29.000000000 +0300
@@ -38,7 +38,7 @@
 .\" AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING
 .\" OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 .\" 
-.TH sasl_getprop "10 July 2001" SASL "SASL man pages"
+.TH sasl_getprop 3 "10 July 2001" SASL "SASL man pages"
 .SH NAME
 sasl_getprop \- Get a SASL property
 
diff -urNad trunk~/man/sasl_getrealm_t.3 trunk/man/sasl_getrealm_t.3
--- trunk~/man/sasl_getrealm_t.3	2006-05-29 22:52:46.000000000 +0300
+++ trunk/man/sasl_getrealm_t.3	2006-07-31 17:59:29.000000000 +0300
@@ -38,7 +38,7 @@
 .\" AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING
 .\" OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 .\" 
-.TH sasl_getrealm_t "10 July 2001" SASL "SASL man pages"
+.TH sasl_getrealm_t 3 "10 July 2001" SASL "SASL man pages"
 .SH NAME
 sasl_getrealm_t \- Realm Acquisition Callback
 
diff -urNad trunk~/man/sasl_getsecret_t.3 trunk/man/sasl_getsecret_t.3
--- trunk~/man/sasl_getsecret_t.3	2006-05-29 22:52:46.000000000 +0300
+++ trunk/man/sasl_getsecret_t.3	2006-07-31 17:59:29.000000000 +0300
@@ -38,7 +38,7 @@
 .\" AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING
 .\" OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 .\" 
-.TH sasl_getsecret_t "10 July 2001" SASL "SASL man pages"
+.TH sasl_getsecret_t 3 "10 July 2001" SASL "SASL man pages"
 .SH NAME
 sasl_getsecret_t \- The SASL callback for secrets (passwords)
 
diff -urNad trunk~/man/sasl_getsimple_t.3 trunk/man/sasl_getsimple_t.3
--- trunk~/man/sasl_getsimple_t.3	2006-05-29 22:52:46.000000000 +0300
+++ trunk/man/sasl_getsimple_t.3	2006-07-31 17:59:29.000000000 +0300
@@ -38,7 +38,7 @@
 .\" AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING
 .\" OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 .\" 
-.TH sasl_getsimple_t "10 July 2001" SASL "SASL man pages"
+.TH sasl_getsimple_t 3 "10 July 2001" SASL "SASL man pages"
 .SH NAME
 sasl_getsimple_t \- The SASL callback for username/authname/realm
 
diff -urNad trunk~/man/sasl_global_listmech.3 trunk/man/sasl_global_listmech.3
--- trunk~/man/sasl_global_listmech.3	2006-05-29 22:52:46.000000000 +0300
+++ trunk/man/sasl_global_listmech.3	2006-07-31 17:59:29.000000000 +0300
@@ -38,7 +38,7 @@
 .\" AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING
 .\" OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 .\" 
-.TH sasl_listmech "10 July 2001" SASL "SASL man pages"
+.TH sasl_listmech 3 "10 July 2001" SASL "SASL man pages"
 .SH NAME
 sasl_listmech \- Retrieve a list of the supported SASL mechanisms
 .SH SYNOPSIS
diff -urNad trunk~/man/sasl_idle.3 trunk/man/sasl_idle.3
--- trunk~/man/sasl_idle.3	2006-05-29 22:52:46.000000000 +0300
+++ trunk/man/sasl_idle.3	2006-07-31 17:59:29.000000000 +0300
@@ -38,7 +38,7 @@
 .\" AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING
 .\" OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 .\" 
-.TH sasl_idle "10 July 2001" SASL "SASL man pages"
+.TH sasl_idle 3 "10 July 2001" SASL "SASL man pages"
 .SH NAME
 sasl_idle \- Perform precalculations during an idle period
 
diff -urNad trunk~/man/sasl_listmech.3 trunk/man/sasl_listmech.3
--- trunk~/man/sasl_listmech.3	2006-05-29 22:52:46.000000000 +0300
+++ trunk/man/sasl_listmech.3	2006-07-31 17:59:29.000000000 +0300
@@ -38,7 +38,7 @@
 .\" AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING
 .\" OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 .\" 
-.TH sasl_listmech "10 July 2001" SASL "SASL man pages"
+.TH sasl_listmech 3 "10 July 2001" SASL "SASL man pages"
 .SH NAME
 sasl_listmech \- Retrieve a list of the supported SASL mechanisms
 .SH SYNOPSIS
diff -urNad trunk~/man/sasl_log_t.3 trunk/man/sasl_log_t.3
--- trunk~/man/sasl_log_t.3	2006-05-29 22:52:46.000000000 +0300
+++ trunk/man/sasl_log_t.3	2006-07-31 17:59:29.000000000 +0300
@@ -38,7 +38,7 @@
 .\" AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING
 .\" OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 .\" 
-.TH sasl_log_t "10 July 2001" SASL "SASL man pages"
+.TH sasl_log_t 3 "10 July 2001" SASL "SASL man pages"
 .SH NAME
 sasl_log_t \- The SASL logging callback
 
diff -urNad trunk~/man/sasl_server_init.3 trunk/man/sasl_server_init.3
--- trunk~/man/sasl_server_init.3	2006-05-29 22:52:46.000000000 +0300
+++ trunk/man/sasl_server_init.3	2006-07-31 17:59:29.000000000 +0300
@@ -38,7 +38,7 @@
 .\" AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING
 .\" OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 .\" 
-.TH sasl_server_init "10 July 2001" SASL "SASL man pages"
+.TH sasl_server_init 3 "10 July 2001" SASL "SASL man pages"
 .SH NAME
 sasl_server_init \- SASL server authentication initialization
 
diff -urNad trunk~/man/sasl_server_new.3 trunk/man/sasl_server_new.3
--- trunk~/man/sasl_server_new.3	2006-05-29 22:52:46.000000000 +0300
+++ trunk/man/sasl_server_new.3	2006-07-31 17:59:29.000000000 +0300
@@ -38,7 +38,7 @@
 .\" AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING
 .\" OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 .\" 
-.TH sasl_server_new "16 May 2001" SASL "SASL man pages"
+.TH sasl_server_new 3 "16 May 2001" SASL "SASL man pages"
 .SH NAME
 sasl_server_new \- Create a new server authentication object
 
diff -urNad trunk~/man/sasl_server_start.3 trunk/man/sasl_server_start.3
--- trunk~/man/sasl_server_start.3	2006-05-29 22:52:46.000000000 +0300
+++ trunk/man/sasl_server_start.3	2006-07-31 17:59:29.000000000 +0300
@@ -38,7 +38,7 @@
 .\" AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING
 .\" OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 .\" 
-.TH sasl_server_start "10 July 2001" SASL "SASL man pages"
+.TH sasl_server_start 3 "10 July 2001" SASL "SASL man pages"
 .SH NAME
 sasl_server_start \- Begin an authentication negotiation
 
diff -urNad trunk~/man/sasl_server_step.3 trunk/man/sasl_server_step.3
--- trunk~/man/sasl_server_step.3	2006-05-29 22:52:46.000000000 +0300
+++ trunk/man/sasl_server_step.3	2006-07-31 17:59:29.000000000 +0300
@@ -38,7 +38,7 @@
 .\" AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING
 .\" OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 .\" 
-.TH sasl_server_step "10 July 2001" SASL "SASL man pages"
+.TH sasl_server_step 3 "10 July 2001" SASL "SASL man pages"
 .SH NAME
 sasl_server_step \- Perform a step in the authentication negotiation
 
diff -urNad trunk~/man/sasl_server_userdb_checkpass_t.3 trunk/man/sasl_server_userdb_checkpass_t.3
--- trunk~/man/sasl_server_userdb_checkpass_t.3	2006-05-29 22:52:46.000000000 +0300
+++ trunk/man/sasl_server_userdb_checkpass_t.3	2006-07-31 17:59:29.000000000 +0300
@@ -38,7 +38,7 @@
 .\" AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING
 .\" OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 .\" 
-.TH sasl_server_userdb_checkpass_t "10 July 2001" SASL "SASL man pages"
+.TH sasl_server_userdb_checkpass_t 3 "10 July 2001" SASL "SASL man pages"
 .SH NAME
 sasl_server_userdb_checkpass_t \- Plaintext Password Verification Callback
 
diff -urNad trunk~/man/sasl_server_userdb_setpass_t.3 trunk/man/sasl_server_userdb_setpass_t.3
--- trunk~/man/sasl_server_userdb_setpass_t.3	2006-05-29 22:52:46.000000000 +0300
+++ trunk/man/sasl_server_userdb_setpass_t.3	2006-07-31 17:59:29.000000000 +0300
@@ -38,7 +38,7 @@
 .\" AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING
 .\" OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 .\" 
-.TH sasl_server_userdb_setpass_t "10 July 2001" SASL "SASL man pages"
+.TH sasl_server_userdb_setpass_t 3 "10 July 2001" SASL "SASL man pages"
 .SH NAME
 sasl_server_userdb_setpass_t \- UserDB Plaintext Password Setting Callback
 
diff -urNad trunk~/man/sasl_setpass.3 trunk/man/sasl_setpass.3
--- trunk~/man/sasl_setpass.3	2006-05-29 22:52:46.000000000 +0300
+++ trunk/man/sasl_setpass.3	2006-07-31 17:59:29.000000000 +0300
@@ -38,7 +38,7 @@
 .\" AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING
 .\" OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 .\" 
-.TH sasl_setpass "10 July 2001" SASL "SASL man pages"
+.TH sasl_setpass 3 "10 July 2001" SASL "SASL man pages"
 .SH NAME
 sasl_setpass \- Check a plaintext password
 .SH SYNOPSIS
diff -urNad trunk~/man/sasl_setprop.3 trunk/man/sasl_setprop.3
--- trunk~/man/sasl_setprop.3	2006-05-29 22:52:46.000000000 +0300
+++ trunk/man/sasl_setprop.3	2006-07-31 17:59:29.000000000 +0300
@@ -38,7 +38,7 @@
 .\" AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING
 .\" OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 .\" 
-.TH sasl_setprop "10 July 2001" SASL "SASL man pages"
+.TH sasl_setprop 3 "10 July 2001" SASL "SASL man pages"
 .SH NAME
 sasl_setprop \- Set a SASL property
 .SH SYNOPSIS
diff -urNad trunk~/man/sasl_user_exists.3 trunk/man/sasl_user_exists.3
--- trunk~/man/sasl_user_exists.3	2006-05-29 22:52:46.000000000 +0300
+++ trunk/man/sasl_user_exists.3	2006-07-31 17:59:29.000000000 +0300
@@ -38,7 +38,7 @@
 .\" AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING
 .\" OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 .\" 
-.TH sasl_user_exists "10 July 2001" SASL "SASL man pages"
+.TH sasl_user_exists 3 "10 July 2001" SASL "SASL man pages"
 .SH NAME
 sasl_user_exists \- Check if a user exists on server
 
diff -urNad trunk~/man/sasl_verifyfile_t.3 trunk/man/sasl_verifyfile_t.3
--- trunk~/man/sasl_verifyfile_t.3	2006-05-29 22:52:46.000000000 +0300
+++ trunk/man/sasl_verifyfile_t.3	2006-07-31 17:59:29.000000000 +0300
@@ -38,7 +38,7 @@
 .\" AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING
 .\" OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 .\" 
-.TH sasl_verifyfile_t "10 July 2001" SASL "SASL man pages"
+.TH sasl_verifyfile_t 3 "10 July 2001" SASL "SASL man pages"
 .SH NAME
 sasl_verifyfile_t \- The SASL file verification
 
